manim/mobject/geometry.py

```diff
@@ -1582,16 +1756,109 @@ def construct(self):
                 self.add(poly_group)
     """
 
-    def __init__(self, n=6, start_angle=None, **kwargs):
-        self.start_angle = start_angle
-        if self.start_angle is None:
-            if n % 2 == 0:
-                self.start_angle = 0
-            else:
-                self.start_angle = 90 * DEGREES
-        start_vect = rotate_vector(RIGHT, self.start_angle)
-        vertices = compass_directions(n, start_vect)
-        Polygon.__init__(self, *vertices, **kwargs)
+    def __init__(self, n: int = 6, **kwargs):
+        super().__init__(n, density=1, **kwargs)
+
+
+class Star(Polygon):
+    """A regular polygram without the intersecting lines.
+
+    Parameters
+    ----------
+    n
+        How many points on the :class:`Star`.
+    outer_radius
+        The radius of the outer vertices.
+    inner_radius
+        The radius of the inner vertices.
+
+        If unspecified, the inner radius will be
+        calculated such that the edges of the :class:`Star`
+        perfectly follow the edges of its :class:`RegularPolygram`
+        counterpart.
+    density
+        The density of the :class:`Star`.
```

Comment:
This explanation should contain the density explanation from `RegularPolygram` since it is a term most users will not understand correctly.